platform/podjs/src/browserPod.ts

```diff
@@ -29,6 +29,11 @@ const NAV_LIGHT_FOREGROUND_COLOR = "#ffffff";
 
 const MANIFEST_DATA = window.manifestData;
 
+/**
+ * It opens a database, creates two object stores for PolyIn and PolyOut keys
+ * and creates a bunch of indexes according to the quad keys.
```

Comment:
Can you please clarify this? All quad keys are indexed?

Reply:
mmh this is what I got - but maybe @snoack has a better saying on this...

Reply:
Well, my point is 
1. What do you mean by PolyIn and PolyOut *keys*?
2. Can you be more precise on "a *bunch* of"?